### ISSUE TITLE
[Disable Feature]: Auto-update of Tonec.InternetDownloadManager

### ISSUE THREAD

**Issue by Dragon1573** [microsoft/winget-pkgs#128340]
## Description

@wingetbot is currently running "version outdated check" automation process for most of the packages. But there's something wrong when it inspecting and submitting new version of package `Tonec.InternetDownloadManager`. In every "auto-update" pull request of that package, `PackageVersion` property is always inspected with [unexpected spaces](https://github.com/microsoft/winget-pkgs/pull/128160/files#diff-447053b625f0b98217103e102c929d2b28dffc37650ca2ad135dcf381c8fed41R5).

In order to avoid further issue with this package, is it possible to disable auto-update of this package? Or is it able to update @wingetbot so it can automatically trim all whitespaces in `PackageVersion`?

- https://github.com/microsoft/winget-pkgs/pull/128160
- https://github.com/microsoft/winget-pkgs/pull/127821

### Proposed technical implementation details (optional)

Nope ...

**Comment by Trenly** [comment]
[Policy] Area-Bots

**Comment by Trenly** [comment]
I don’t believe it is possible to turn it off for specific packages; However, I do agree with the idea to trim whitespace

**Comment by stephengillie** [comment]
I've added a note for this `PackageIdentifier` to my Review list. It's currently only partially functional, but at a minimum it will remind me and help quicken damage control.

**Comment by Dragon1573** [comment]
> I would like to note that this should be mitigated once Winget v1.9 is released, as spaces are trimmed from each version part in 1.9

The current latest stable release is version [`1.9.25200`](https://github.com/microsoft/winget-cli/releases/tag/v1.9.25200). This issue should be already resolved.
